openfl/interface/aggregator.py

```diff
@@ -36,7 +36,13 @@ def aggregator(context):
         default='plan/cols.yaml', type=ClickPath(exists=True))
 @option('-s', '--secure', required=False,
         help='Enable Intel SGX Enclave', is_flag=True, default=False)
-def start_(plan, authorized_cols, secure):
+@option('-c', '--cert_path',
```

Comment:
What if we allow user to provide paths to certificates and keys? This way we do not implicitly rely on the user certificate folder to replicate our internal structure. For example, we will not expect aggregator private key be in format `{CERT_DIR}/server/agg_{common_name}.key`

Reply:
We should not force users to keep our internal PKI folder structure. I think it is better to split this argument and use paths for each file.